app/auth/auth.py

```diff
@@ -1,21 +1,20 @@
 from app import login_manager, bcrypt, db
-from app.database.database import Users, UserIncomeCategory, UserOutcomeCategory, OutcomeCategory, IncomeCategory
+from app.database.database import Users, Groups, Category, Subcategory, UserGroup, Transactions, Goals, Budget
 
 
 def insert_user(username, password):
     hashed_password = bcrypt.generate_password_hash(password).decode("utf-8")
     user = Users(user_name=username, password=hashed_password)
     db.session.add(user)
-    db.session.commit()
+    db.session.commit()  # co to robi?
```

Comment:
Yuo can delete flush and refresh (flush is included in commit). I think refres is not needed. I checked that users and categorys are properly added without this comends.